app/src/main/javascript/bundles/experiment-page/src/TabConfig.js

```diff
@@ -0,0 +1,28 @@
+
+// Helper function to check if an object is empty
+export const isObjectEmpty = (objectName) => {
+    return (
+        objectName &&
+        Object.keys(objectName).length === 0 &&
+        objectName.constructor === Object
+    );
+};
+
+export const isEmptyArray = (value) => isObjectEmpty(value) && Array.isArray(value);
+
+export const tabValidations = new Map([
```

Comment:
You can use `object` data type for `tabValidations`, `tabCommonValidations` and `innerTabValidations`, which is more common and easier for implementing and understanding.